src/server/exotics/task-list-incomplete/index.js

```diff
@@ -7,6 +7,8 @@ const pageHeadingAndTitle =
   'You need to complete all of the sections before you review and submit'
 
 export class TaskListIncompletePage extends Page {
+  key = 'taskListIncomplete'
+  sectionKey = 'exotics'
```

Comment:
Are these only used for namespacing for the purposes of the plugin name? 🤔

Reply:
yes, if we don't have them then we are registering a plugin with name undefined-undefined and there can only be one, so I had a conflict with your task list incomplete page and had to fix it